tests/python/base_test.py

```diff
@@ -46,13 +46,6 @@ def setUp(self):
         # now make a context
         context = self.tk.context_from_entity(self.project["type"], self.project["id"])
 
-        patched = patch(
```

Comment:
I've removed this mock. Let's assume Qt is always available, like on our CI instances, and then we can use Qt in our tests.